SwiftTweaks/UIImage+SwiftTweaks.swift

```diff
@@ -25,7 +25,11 @@ internal extension UIImage {
 	// NOTE (bryan): if we just used UIImage(named:_), we get crashes when running in other apps!
 	// (Why? Because by default, iOS searches in your app's bundle, but we need to redirect that to the bundle associated with SwiftTweaks
 	private convenience init?(inThisBundleNamed imageName: String) {
+		#if SWIFT_PACKAGE
+		self.init(named: imageName, in: Bundle.module, compatibleWith: nil)
```

Comment:
[`Bundle.module` is available within a Swift Package](https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package#3578939)